metadata-db-app/src/cmr/metadata_db/data/oracle/granule_table.clj

```diff
@@ -4,13 +4,7 @@
    [clojure.java.jdbc :as jdbc]
    [cmr.metadata-db.data.util :as util]))
 
-(defmulti granule-column-sql
```

Comment:
This is just a curiosity question. But are we trying to remove defmulti because it's hard to follow or not recommended design?

Reply:
I read a while ago online - I can't find it now though, as well as from a previous CMR clojure developer who really knew clojure - that defmulti is slow processing wise.  I can't quite remember the numbers but it is something like 20 to 100 times slower than using other branching mechanisms. We really don't need polymorphism in these instances and in a lot of cases, I seem to be able to reduce code.

Reply:
20-100x slower?! wow.

Reply:
I can't find the exact number but when I read it, it was significant.  Since I can't find that article anymore, I also can't tell you how long ago it was written.  It is possible that clojure has fixed up the slowless using multidef, but not sure.  I could create a test and see, but I am finding that the code becomes cleaner when I don't use multidef  (and when I take the time to properly refactor the defmethods).

Reply:
I personally don't like multidef because it is difficult to follow, so I am onboard with this change